index.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
 
   env: {
     es6: true,
+    jest: true,
```

Comment:
Yeah, this is what we are doing at the moment. The idea is to move to `jest` in general anyway.
But it doesn't matter much if some flags are defined here (e.g. `mocha` is also defined here, although it's not used in every project).

I don't have a strong opinion on this, I'm fine with what we decide.